src/GovUk.Education.ExploreEducationStatistics.Content.Services/ThemeService.cs

```diff
@@ -164,8 +165,8 @@ private async Task<PublicationTreeNode> BuildPublicationNode(Publication publica
             {
                 case PublicationTreeFilter.FindStatistics:
                     return !publicationTreeNode.IsSuperseded
-                           && (publicationTreeNode.LatestReleaseHasData ||
-                               !string.IsNullOrEmpty(publicationTreeNode.LegacyPublicationUrl));
+                           && (publicationTreeNode.HasLiveRelease
```

Comment:
Could this be simplified to `&& (publicationTreeNode.HasLiveRelease || publicationTreeNode.Type == Legacy)`?

Reply:
I've changed it. I've also changed a similar bit of code in `ThemeService#BuildTopicTree`.